projects/media-viewer/src/lib/media-viewer/model/toolbar-button-toggles.ts

```diff
@@ -0,0 +1,55 @@
+import { BehaviorSubject } from 'rxjs';
+
+export const getToolbarButtonToggles = (contentType: string) => {
+  switch (contentType) {
+    case 'pdf':
+      return  new PdfViewerToolbarButtons();
+      break
```

Comment:
if you are returning then you don't need a break.... I know you probably just left them in for consistency but it seems a bit odd. If you want to keep it at least at a semi-colon - we're not heretics after all :)